README.md

```diff
@@ -53,14 +56,48 @@ If you need a more custom setup, define the services in your application configu
 
 ```yaml
 httplug:
-    main_alias:
-        client: httplug.client.default
-        message_factory: httplug.message_factory.default
-        uri_factory: httplug.uri_factory.default
-    classes:
-        client: ~ # uses discovery if not specified
-        message_factory: ~
-        uri_factory: ~
+  main_alias:
+    client: httplug.client.default
+    message_factory: httplug.message_factory.default
+    uri_factory: httplug.uri_factory.default
+    stream_factory: httplug.stream_factory.default
+  classes:
+    # uses discovery if not specified
+    client: ~
+    message_factory: ~ 
+    uri_factory: ~
+    stream_factory: ~
+```
+
+#### Configure your client
+
+You can configure your clients with some good default options. The clients are later registered as services. 
+
+```yaml
+httplug:
+  clients: 
+    my_guzzle5: 
+      adapter: guzzle5
```

Comment:
if this would be the full factory service name instead of a name for the meta-factory, we could get rid of the the service tags and the meta factory that registers factories by short name (aka is a mini-container for httplug factories)